packages/lisk-transactions/test/transactions/7_out_transfer_transaction.ts

```diff
@@ -13,21 +13,27 @@
  *
  */
 import { expect } from 'chai';
+import { MockStateStore as store } from '../helpers';
 import {
 	OutTransferTransaction,
-	Attributes,
-	BaseTransaction,
 } from '../../src/transactions';
 import { validOutTransferTransactions } from '../../fixtures';
 import { Status, TransactionJSON } from '../../src/transaction_types';
 
 describe('outTransfer transaction class', () => {
 	const defaultTransaction = validOutTransferTransactions[0];
+	const defaultValidSender = {
+		address: '13155556493249255133L',
+		balance: '500010000000',
+		publicKey:
+			'305b4897abc230c1cc9d0aa3bf0c75747bfa42f32f83f5a92348edea528850ad',
+	};
 
 	let validTestTransaction: OutTransferTransaction;
 
 	beforeEach(async () => {
 		validTestTransaction = new OutTransferTransaction(defaultTransaction);
+		store.account.get = () => defaultValidSender;		
```

Comment:
I think it would be better if we use stub for store and check the input too, but let's create another issue for that and tackle it separately